liferay-echo/liferay-portal
# [TS] LPS-172057

Motivation
=======
This fix solves case described in https://issues.liferay.com/browse/LPS-172057

Proposed Solution
========
Apply the same formatting logic of NumericDDMFormFieldUtil in TemplateNode.

Steps to Verify
========
Steps to reproduce:

1. Create a new structure and add a Numeric (integer) field.
2. Add a template for 'test' displaying the field.
3. Create a new 'test' content, and fill the value: 8
4. Save as draft.
5. Add the 'ar' locale to the URL, so the edition page is displayed in arabic.
6. Visualize the arabic content through the template (through the "eye" icon)
